mne/conftest.py

```diff
@@ -140,6 +140,10 @@ def pytest_configure(config):
     ignore:Implementing implicit namespace packages.*:DeprecationWarning
     ignore:Deprecated call to `pkg_resources.*:DeprecationWarning
     ignore:pkg_resources is deprecated as an API.*:DeprecationWarning
+    # TODO: This one we should fix with .item()s
+    ignore:Conversion of an array with ndim > 0 to a scalar.*:DeprecationWarning
+    # h5py
+    ignore:`product` is deprecated as of NumPy.*:DeprecationWarning
```

Comment:
same here, I see some `product` -> `prod` in the diff.

Reply:
The `h5py` comment above is meant to indicate that it's (now) h5py's problem rather than ours